graphql/shop/helpers.ts

```diff
@@ -0,0 +1,51 @@
+import { ShopDetails } from './../../graphql.d';
+import { ShopDetails as PrismaShopDetails } from '@prisma/client';
+
+import {
+  nowFromCoordinates,
+  todayIs,
+  isOpen as IsOpenHelper,
+} from 'src/utils/dates';
+import { parseUTCTime } from 'src/utils/dates';
+import { parsePhone } from 'src/utils/phone-utils';
+
+const todaysStatus = (shopDetails: ShopDetails | PrismaShopDetails) => {
```

Comment:
Why both types?

Reply:
it can be called from a SSR request and be a direct Prisma db entity or from a graphql resolver, so the helper should handle both cases